github/pull_request.go

```diff
@@ -153,9 +152,9 @@ func (p *pullRequestContent) OnlyFreebsd() bool {
 	var linux bool
 
 	for _, f := range p.files {
```

Comment:
I was thinking the code *should* be something like:

```go
for _, f := range p.files {
    if !isFreeBSD(f.FileName) {
        return false
    }
}
return true
```

And almost the same for `OnlyWindows()` below..